EasyPost/ClientConfiguration.cs

```diff
@@ -120,4 +120,37 @@ internal void SetUp()
 #pragma warning disable CA1307
     public override int GetHashCode() => ApiKey.GetHashCode() ^ ApiBase.GetHashCode() ^ Timeout.GetHashCode();
 #pragma warning restore CA1307
+
+    private bool _isDisposed;
+
+    public void Dispose()
```

Comment:
suggestion: would love docstrings for these. Although "dispose" is pretty easy to follow, what exactly that does at runtime, the compiler isn't well known to everyone and could be documented.

Reply:
Will do in the docstrings PR.